.vscode/cspell.json

```diff
@@ -1385,6 +1385,24 @@
       "words": [
         "AISEARCH"
       ]
+    },
+    {
+      "filename": "sdk/onlineexperimentation/**",
+      "words": [
+        "momo",
+        "createorupdatemetric",
+        "createorupdatemetricaverage",
+        "createorupdatemetriceventcount",
+        "createorupdatemetriceventrate",
+        "createorupdatemetricpercentile",
+        "createorupdatemetricsum",
+        "createorupdatemetricusercount",
+        "createorupdatemetricuserrate",
+        "vxexdn",
+        "ijchzdh",
```

Comment:
Some of it comes from generated sample/test.

These files are to help you write your own sample/test. If you write it, you can delete the "generated" folder and get rid of these suppress.
https://github.com/Azure/azure-sdk-for-java/wiki/TypeSpec-Java-Quickstart#samples

Reply:
Yes, you can delete the `generated` folder. It won't cause build problem.

Reply:
@weidongxu-microsoft, I looked through other SDKs, it looks like a good number have keep both generated samples as well as custom made ones. Are you OK to keep this as-is or should I remove them?

Reply:
Cleaned up the generated tests & samples, and reduced cspell.json.

Reply:
Thanks. You can use e.g. `tsp-client update --emitter-optoins="generate-tests=false"`, if you don't want them pop up in next codegen.

You can also do same to turn it off in tspconfig.yaml. But if some of the test/sample helps this time, you may want to keep them there, as in next api-version they might help too.

We likely can have a better sample id in future codegen. Here your JSON `CreateOrUpdateMetric_Average.json` becomes `createorupdatemetricaverage`. Codegen probably better produce `create-or-update-metric-average`.

Yeah, some of existing lib have both co-exist. But that mostly for service not writing enough samples. I see here you already had quite many.